scripts/build_dependencies.sh

```diff
@@ -13,10 +13,12 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 target=Debug
 source $DIR/_init.sh $@
 source $DIR/build_openssl.sh -t $target
-source $DIR/build_curl.sh    -t $target
-source $DIR/build_awssdk.sh  -t $target
-source $DIR/build_cmocka.sh  -t Debug
+source $DIR/build_oob.sh     -t $target
```

Comment:
ah ok, this was missing.

Reply:
ah, you know what, this is no longer used in the CI.
```
ci/build/build.sh
```

Reply:
Ah, I see. Does that script also work for local development? Also the README should be updated

Reply:
Yeah, this is to start the build.
```
./ci/build_linux.sh
```
set `NEXUS_PASSWORD` to the ldap password.